about.html

```diff
@@ -0,0 +1,182 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>About | Dev Conference 2023</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700;900&display=swap" rel="stylesheet">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha1/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-GLhlTQ8iRABdZLl6O3oVMWSktQOp6b7In1Zl3/Jr59b6EGGoI1aFkw7cmDA6j6gD" crossorigin="anonymous">
+    <link rel="stylesheet" href="./assests/css/styles.css">
+</head>
+
+<body>
+    <header>
+        <div class="headline">
+            <div class="container">
+                <div class="d-flex ms-auto">
+                    <span>
+                        <i class="fa-brands fa-facebook-f"></i>
+                    </span>
+                    <span>
+                        <i class="fa-brands fa-twitter"></i>
+                    </span>
+                    <span>English</span>
+                    <span>My page</span>
+                    <span>Logout</span>
+                </div>
+            </div>
```

Comment:
- [ ] The top dark bar of your application is good, but on the mobile version, it should be hidden. Kindly hide this navbar to match the template.
 - Your version:
 - 
<img width="376" alt="image" src="https://user-images.githubusercontent.com/61919617/217869465-182b37b4-1ab4-41a4-b794-37b7755334be.png">

 - Template:
 - 
<img width="405" alt="image" src="https://user-images.githubusercontent.com/61919617/217869561-1c1213f4-d89e-48a0-93a5-de90584b73da.png">